scenes/novoProjeto.scene.js

```diff
@@ -1,77 +1,88 @@
-const { User } = require('../models/user.model');
-const Scene = require('telegraf/scenes/base')
+/* eslint-disable no-plusplus */
+/* eslint-disable func-names */
+/* eslint-disable no-underscore-dangle */
+const winston = require('winston');
+const Scene = require('telegraf/scenes/base');
 const { Markup } = require('telegraf');
 const axios = require('axios');
-const winston = require('winston');
+const { User } = require('../models/user.model');
 
 const novoProjetoScene = new Scene('novoprojeto');
 
-let projeto = {};
+const projeto = {};
 
-novoProjetoScene.enter(async (ctx) => {
-    return ctx.replyWithMarkdown(`Vamos lá! Para começar o cadastro do seu projeto, primeiramente me informe o ID do mesmo.`);
-})
+const setwebHook = async function (projectId, token) {
+  const hookUrl = 'http://51.79.87.65:3000/api/gitlabIntegration';
+  let alreadyRegistered = false;
 
-novoProjetoScene.on('text', async (ctx) => {
-    projeto._id = ctx.message.text;
-    let user = await User.findById(ctx.message.from.id);
-    let projetoFromDb = user.projects.find(proj => {
-        return proj._id == projeto._id;
-    })
-    if (projetoFromDb) return ctx.reply('O projeto já está cadastrado! Por favor, tente novamente.');
-    let res = await axios.get('https://gitlab.com/api/v4/projects/'+projeto._id, { headers: { 'Private-Token': user.token } });
-    projeto.nome = res.data.name;
-    return ctx.reply(
-        `Apenas confirmando, o projeto em questao é o ${res.data.name}?`,
-        Markup.inlineKeyboard([Markup.callbackButton('Sim, é este projeto', 'yes'), Markup.callbackButton('Não, não é este', 'no')]).extra()
-    )
-})
+  const res = await axios.get(`https://gitlab.com/api/v4/projects/${projectId}/hooks`, { headers: { Authorization: `Bearer ${token}` } });
+  for (let index = 0; index < res.data.length; index++) {
+    if (res.data[index] === hookUrl) {
+      alreadyRegistered = true;
+      break;
+    }
+  }
```

Comment:
Aqui você pode utilizar a função [`.some( )`](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Array/some) para percorrer o array e verificar se pelo menos um dos itens de `res.data` é igual a `hookUrl`:

```js
conts alreadyRegisted = res.data.some(value => value === hookUrl)
```

O efeito é o mesmo e no final você terá uma solução mais 'elegante' no lugar da utilização de `for break loop`

Reply:
:open_mouth: pior q eu pesquisei ainda 'check if an array contains a value javascript' mas recebi uns resultados nada a ver no google. E esse codigo ainda ta errado porque eu deveria comparar `data[index].url `nao apenas `data[index]` mas ja foi corrigido 90b4c03bc13ceab90001103f5fc03bcf9b349926

Reply:
Google in a nutshell:

me: nodejs how to print in console 

google: You need to create a instance in AWS EC2, deploy a docker container by using Kubernetes, add babel with webpack in your project, write awesome Design Structured Application with MCV Architecture, comunicating with mongodb, read the message from the database and then, write a json structured log using Pino, sending it to kibana with grafana pluged in to count how many times "hello world" was printed in terminal in every 5 minutes.